ui/FilesPanel.go

```diff
@@ -246,15 +248,11 @@ func (this *filesPanel) createRootLocationButton(location dataModels.Location) *
 
 	rootLocationButton, _ := gtk.ButtonNew()
 	rootLocationButton.Connect("clicked", func() {
```

Comment:
The existing impl is essentially:
```
click handler {
	this.UI.sdNotify(daemon.SdNotifyWatchdog)
	doLoadFiles() {
		EmptyTheContainer()
		getSortedFiles() {
			...
		}
		addSortedFiles()
		listBox.ShowAll()
	}
	this.UI.sdNotify(daemon.SdNotifyReady)
}
```

And your change moves the call with SdNotifyWatchdog to inside getSortedFiles():
```
click handler {
	doLoadFiles() {
		EmptyTheContainer()
		getSortedFiles() {
			this.UI.sdNotify(daemon.SdNotifyWatchdog)
			...
		}
		addSortedFiles()
		listBox.ShowAll()
	}
}
```

1) Why move sdNotify(daemon.SdNotifyWatchdog) to getSortedFiles()?  Why not wrap everything?
2) SdNotifyWatchdog is called, but not SdNotifyReady.  Doesn't there need to be a corresponding/matching call to sdNotify() with SdNotifyReady?

Reply:
Sounds good, thanks.